### DISCUSSION TITLE
Template

### DISCUSSION THREAD

**Discussion by David5886** [cs481-ekh/f22-fires-wild#1] [Meeting Minutes]
[ date if not submitted on the day of the meeting ]
Excused: [ list of those not in attendance who notified the group ]
Absent: [ list of those not in attendance who did not notify the group ]

Summary: [ summarize the topics that were discussed ]

Action Items: [ list of action times by assignee ]

**Comment by jeremystocking** [comment]
Fires Wild
09/04/2022
Excused: 0
Absent: 0
Summary: 
	The majority of our conversation was about which frameworks/languages we everyone would like to use. We discussed some of the challenges we may face as we are becoming more familiar with the project. The remainder of our meeting involved us finishing the required sections of our project plan. We were able to assign everyone a task to work on before our next meeting listed under action items. 
Action Items: 
1) Jeremy- Connect to a test database via localhost using PHP/MyPHPAdmin.
2) Benji-Connect to a test database via localhost using Python/DjangoDB.
3) David- Familiarity with the data and getting the entire data set into a database.
4) Sam- Familiarity with Leaflet and a “hello world" map.
5) Brenden- Familiarity with React and some of the front-end challenges we may face.
